readyplayerme/rpm-unity-sdk-avatar-creator
# [SDK-409] Add modular assets

## [SDK-409](https://ready-player-me.atlassian.net/browse/SDK-409)

## Description

- Refactored category buttons and panel creation
- Added modular assets category
- Refactored AvatarAPIRequests
- Added TemplateFetcher

## How to Test

- Play and check the modular assets, try all categories, all body type, all genders.



## Checklist








[SDK-409]: https://ready-player-me.atlassian.net/browse/SDK-409?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
